models/app.ts

```diff
@@ -0,0 +1,41 @@
+export interface App {
```

Comment:
nice 👍

Reply:
Best part about using TypeScript across the stack, at some point you can share models between the backend/frontend